vars/RedHat.yml

```diff
@@ -13,7 +13,7 @@ metricbeat_repo_gpgkey: https://artifacts.elastic.co/GPG-KEY-elasticsearch
 metricbeat_baseurl: "https://download.elastic.co/beats/metricbeat"
 
 metricbeat_packages:
-  - "metricbeat"
+  - "metricbeat{{ (metricbeat_upgrade) | ternary('', '-' + metricbeat_version) }}"
```

Comment:
have you tried this succesfully? this was done this way since the official elastic repos don't include the version, only the latest version is available

Reply:
It looks like the yum repo with baseurl : https://artifacts.elastic.co/packages/5.x/yum has some history

```
# yum list metricbeat --showduplicates --disablerepo=* --enablerepo elasticsearch-5.x
Modules complémentaires chargés : fastestmirror
Loading mirror speeds from cached hostfile
Paquets installés
metricbeat.x86_64                               5.5.2-1                                @elasticsearch-5.x
Paquets disponibles
metricbeat.i686                                 5.0.0-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.0.0-1                                elasticsearch-5.x 
metricbeat.i686                                 5.0.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.0.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.0.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.0.2-1                                elasticsearch-5.x 
metricbeat.i686                                 5.1.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.1.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.1.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.1.2-1                                elasticsearch-5.x 
metricbeat.i686                                 5.2.0-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.2.0-1                                elasticsearch-5.x 
metricbeat.i686                                 5.2.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.2.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.2.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.2.2-1                                elasticsearch-5.x 
metricbeat.i686                                 5.3.0-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.3.0-1                                elasticsearch-5.x 
metricbeat.i686                                 5.3.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.3.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.3.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.3.2-1                                elasticsearch-5.x 
metricbeat.i686                                 5.3.3-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.3.3-1                                elasticsearch-5.x 
metricbeat.i686                                 5.4.0-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.4.0-1                                elasticsearch-5.x 
metricbeat.i686                                 5.4.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.4.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.4.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.4.2-1                                elasticsearch-5.x 
metricbeat.i686                                 5.4.3-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.4.3-1                                elasticsearch-5.x 
metricbeat.i686                                 5.5.0-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.5.0-1                                elasticsearch-5.x 
metricbeat.i686                                 5.5.1-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.5.1-1                                elasticsearch-5.x 
metricbeat.i686                                 5.5.2-1                                elasticsearch-5.x 
metricbeat.x86_64                               5.5.2-1                                elasticsearch-5.x 
```

Reply:
yes, but the name of the package is always duplicated, how would you differentiate to install an old version if `metricbeat_pacakge` is always `metricbeat` ?

Reply:
I'm confused about your question.
Is it about Ansible code or about the log?

For the log, there are 2 different packages for every version depending on  the CPU architecture.:
- x86_64 suffix indicates a 64bit processor
- i686 a 32bit one

For the code, I tried to explain in the description of the PR.
* If we use ``` metricbeat_upgrade: true``` then no need to specify the version number because we want the latest one available on the repos. So ```metricbeat_pacakge``` stays equal to ```metricbeat```
* On the contrary, if we use ``` metricbeat_upgrade: false``` then we add the version number into the package name (as yum and alike expects it). So ```metricbeat_pacakge``` looks like ```metricbeat-5.5.2```.

That's a standard way to use rpm package manager either:
* ask for the latest one ```yum install httpd```
* ask for a particular version ```yum install httpd-2.2.15```